applications/ULFapplication/ULF_application.cpp

```diff
@@ -1,7 +1,7 @@
 //
 //   Project Name:        Kratos
 
-//   Last Modified by:    $Author: virginia $
```

Comment:
The new headers are like:

~~~c
//    |  /           |
//    ' /   __| _` | __|  _ \   __|
//    . \  |   (   | |   (   |\__ \.
//   _|\_\_|  \__,_|\__|\___/ ____/
//                   Multi-Physics
//
//  License:		 BSD License
//					 Kratos default license: kratos/license.txt
//
//  Main authors:    Riccardo Rossi
//  Colaborator:     Vicente Mataix Ferrandiz
//
~~~

Reply:
thank you Vicente. Is there any way of making this change automatically or do I have to do it by manual editing?

Reply:
Unfortunately you should do this by hand, at least what I know

Reply:
@playpaolo it is encouraged to do it but you can postponed it to next commits if you like.

Reply:
Just to add that right now is more important to have a compilable ulf app as soon as possible.

Reply:
Okay, so I will start by correcting the rest of things